src/services/indexer/indexer.ts

```diff
@@ -108,6 +108,7 @@ export async function handleTransactions(db: MyDatabase, tonApi: AxiosInstance,
                         extra.loadUint(64); // protocol gift
                         extra.loadUintBig(256); // collateral asset id
                         extra.loadUintBig(64); // delta collateral principal
+                        extra.loadUintBig(64); // something related to collateral, but it's not the actual returned during a liquidation
                         const collateralAmount = extra.loadUintBig(64);
```

Comment:
```suggestion
                        extra.loadUintBig(64); // collateralAmount
                        const liquidatorGane = extra.loadUintBig(64);
```

Reply:
Yeah, it's more understandable variable name. Anyway, mainstream already updated with the right fix. Let's close this PR.